kubernetes/onpremise/kubeadm/examples/complete/main.tf

```diff
@@ -0,0 +1,49 @@
+
+locals {
+  token = "${random_string.token_id.result}.${random_string.token_secret.result}"
+}
+
+# call the module for kubeadm cluster creation
+module "install_kubeadm_cluster" {
+  source = "../../../kubeadm"
+
+  master_public_ip            = var.master.public_dns
+  master_private_ip           = var.master.private_dns
+  tls_private_key_pem_content = file(var.master.tls_private_key_pem_file)
+  user                        = var.user
+  kubeadm_token               = local.token
+  cni_pluggin                 = "flannel" # calico or flannel
```

Comment:
Did you use Terraform validation for variable `cni_pluggin` to declare the valid values `calico` and `flannel`?